Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -3103,39 +3104,10 @@ void qMRMLSegmentEditorWidget::onSelectSegmentShortcut()
     {
     return;
     }
-  int segmentIndexOffset = shortcut->property("segmentIndexOffset").toInt();
 
-  vtkMRMLSegmentationNode* segmentationNode = vtkMRMLSegmentationNode::SafeDownCast(this->segmentationNode());
-  if (segmentationNode == nullptr || segmentationNode->GetDisplayNode() == nullptr)
-    {
-    return;
-    }
-  vtkMRMLSegmentationDisplayNode* displayNode = vtkMRMLSegmentationDisplayNode::SafeDownCast(segmentationNode->GetDisplayNode());
-  if (displayNode == nullptr)
-    {
-    return;
-    }
-  std::vector<std::string> visibleSegmentIDs;
-  displayNode->GetVisibleSegmentIDs(visibleSegmentIDs);
-  QString currentSegmentID = this->currentSegmentID();
-  QSet<QString> visibleSegmentIDsSet;
-  for (std::string visibleSegmentID : visibleSegmentIDs)
-    {
-    visibleSegmentIDsSet << QString::fromStdString(visibleSegmentID);
-    }
-  QSet<QString> displayedSegmentIDsSet = d->SegmentsTableView->displayedSegmentIDs().toSet();
-  QStringList iterableSegmentIds = visibleSegmentIDsSet.intersect(displayedSegmentIDsSet).toList();
-  for (int segmentIndex = 0; segmentIndex < iterableSegmentIds.size(); segmentIndex++)
-    {
-    QString segmentID = iterableSegmentIds[segmentIndex];
-    if (currentSegmentID == segmentID)
-      {
-      // this is the current segment, determine which one is the previous/next and select that
-      int newSegmentIndex = (segmentIndex + segmentIndexOffset) % iterableSegmentIds.size(); // wrap around
-      this->setCurrentSegmentID(iterableSegmentIds[newSegmentIndex].toStdString().c_str());
-      return;
-      }
-    }
+  // Direction that the segment selection should be moved (-ve/+ve is previous/next)
```

Comment:
What is the meaning of `ve` ?

Reply:
Should the comment be `-1/+1 is previous/next` ?

Reply:
Yeah, it should be -1/+1. Changed.